Готовое задание

VL-SL opened the pull request.

VL-SL committed:
CI/CD

VL-SL committed:
Completed task

VL-SL committed:
Попытка 2

VL-SL committed:
Попытка 3

VL-SL committed:
Попытка 4

malov-rrc commented on src/test/java/svm/simbirsoft/pojo/Entity.java:
```diff
@@ -0,0 +1,27 @@
+package svm.simbirsoft.pojo;
```
Мелочь, но всё таки: обычно models такую папку называют)

malov-rrc commented on src/test/java/svm/simbirsoft/tests/EntityCreateTest.java:
```diff
@@ -0,0 +1,49 @@
+package svm.simbirsoft.tests;
+
+import io.qameta.allure.*;
```
Wildcard импорт убираем, проверь везде

malov-rrc commented on src/test/java/svm/simbirsoft/tests/EntityCreateTest.java:
```diff
@@ -0,0 +1,49 @@
+
+        Response response = BaseRequests.post("/api/create", requestEntity);
+
+        Assert.assertEquals(response.getStatusCode(), 200, "Сущность была не добавлена (неверный статус код)" );
```
Так тоже ок, но: эту проверку можно ещё силами самого rest-assured'а осуществить, давай в некоторых местах попробуем (для общего развития)

VL-SL committed:
1. Изменено название папки pojo на models.
2. Убран Wildcard импорт
3. Параллельность по методам
4. В DeleteTest проверки реализованы с помощью самого rest-assured'а
